server/index.js

```diff
@@ -92,6 +101,28 @@ app.get('/api/v1/search', (req, res) => {
   })
 })
 
+app.put('/api/v1/me/player/play', jsonParser,(req, res) => {
+  const access_token = req.cookies.SPOTIFY_ACCESS_TOKEN
+  
+  // const search_query_parameters = new URLSearchParams({
+  //   ...req.query
+  // })
+  //Get device_id to component and send through query parameter to store in playOptions.
```

Comment:
I'm not sure how helpful this comment is in describing what we're doing in the `PUT /api/v1/me/player/play` endpoint. Consider adding a more descriptive comment explaining the expected query parameters and body of requests made to `PUT /api/v1/me/player/play`

Reply:
Thanks for the comment removal detection service!